docs/network-flow-visibility.md

```diff
@@ -1,12 +1,101 @@
+and are sent to the configured flow controller. High-level design is given below:
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/netviz.png" width="600" alt="Network Flow Visibilty">
+
+## Flow Exporter feature
+
+In Antrea, the basic building block for the Network Flow Visibility is the **Flow Exporter feature**. Flow Exporter operates
+within Antrea Agent; it builds and maintains a connection store by polling and dumping flows from conntrack module periodically.
+Connections from the connection store are exported to a flow collector using the IPFIX protocol, and for this purpose we
+use the [go-ipfix](https://github.com/vmware/go-ipfix) library.
+ 
+### Configuration
+
+To enable the Flow Exporter feature at the Antrea Agent, the following config parameters have to be set in the Antrea Agent
+ConfigMap as shown below. We provide some examples for the parameter values in the following snippet.
+
+```yaml
+  antrea-agent.conf: |
+    # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+    featureGates:
+    # Enable flowexporter which exports polled conntrack connections as IPFIX flow records from each agent to a configured collector.
+      FlowExporter: true
+    # Enable antrea proxy which provides ServiceLB for in-cluster services in antrea agent.
+    # It should be enabled on Windows, otherwise NetworkPolicy will not take effect on
+    # Service traffic.
+      AntreaProxy: true
+
+    # Provide flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp. This also enables
+    # the flow exporter that sends IPFIX flow records of conntrack flows on OVS bridge. If no L4 transport proto is given,
+    # we consider tcp as default.
+    flowCollectorAddr: "192.168.86.86:4739:tcp"
+
+    # Provide flow poll interval as a duration string. This determines how often the flow exporter dumps connections from the conntrack module.
+    # Flow poll interval should be greater than or equal to 1s (one second).
+    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+    flowPollInterval: "1s"
+
+    # Provide flow export frequency, which is the number of poll cycles elapsed before flow exporter exports flow records to
+    # the flow collector.
+    # Flow export frequency should be greater than or equal to 1.
+    flowExportFrequency: 5
+```
+ 
+Please note that the default values for `flowPollInterval` and `flowExportFrequency` parameters are set to 5s and 12, respectively.
+`flowCollectorAddr` is a required parameter that is necessary for the Flow Exporter feature to work.
+
+### Supported capabilities
+
+Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Node, Node-to-Pod, Node-to-Node and Pod-to-Service
+network flows along with the associated statistics such as byte throughput, packet throughput, cumulative byte count, cumulative packet
```

Comment:
"byte throughput, packet throughput" 
I understand the meanings given the context. Are these commonly used terms though? I do see we have "Node Throughput" below and these two could confuse with it.

Reply:
Hi Kai,
I see that packet throughput to be widely used, however byte throughput may be considered informal. Changed it to data throughput and also added units to avoid confusion with metrics shown under Node Throughput.